src/nrepl/connectSequence.ts

```diff
@@ -208,7 +209,10 @@ function getConnectSequences(projectTypes: string[]): ReplConnectSequence[] {
     const customSequences = getCustomConnectSequences();
     const defSequences = projectTypes.reduce((seqs, projectType) => seqs.concat(defaultSequences[projectType]), []);
     const defSequenceProjectTypes = [...new Set(defSequences.map(s => s.projectType))];
-    const sequences = customSequences.filter(customSequence => defSequenceProjectTypes.includes(customSequence.projectType)).concat(defSequences);
+  const sequences = customSequences.filter(customSequence =>
+      !!customSequence.projectRootDir
+      || defSequenceProjectTypes.includes(customSequence.projectType)
+  ).concat(defSequences);
```

Comment:
I don't quite remember where the result from this function is used. Can you explain why you filter away sequences with the setting? (I'm sure it is the right thing to do, just want to understand why. 😄 )